src/types.ts

```diff
@@ -384,6 +385,26 @@ export interface EarlyAccessFeatureResponse {
     earlyAccessFeatures: EarlyAccessFeature[]
 }
 
+/** A user's support tickets */
+export interface SupportTicket {
+    // Sync this with the backend's SupportTicketSerializer!
+    id: number
+    created_at: Date
+    updated_at: Date
+    subject: string
+    description: string
+    status: 'open' | 'closed'
```

Comment:
there's three different statuses we need here - input needed (need input from user, internally this is pending), in progress (waiting for support), closed